packages/desktop-client/src/components/schedules/StatusBadge.tsx

```diff
@@ -9,62 +9,71 @@ import CheckCircleHollow from '../../icons/v2/CheckCircleHollow';
 import EditSkull1 from '../../icons/v2/EditSkull1';
 import FavoriteStar from '../../icons/v2/FavoriteStar';
 import ValidationCheck from '../../icons/v2/ValidationCheck';
-import { colors, type CSSProperties } from '../../style';
+import { theme, type CSSProperties } from '../../style';
 import Text from '../common/Text';
 import View from '../common/View';
 
 export function getStatusProps(status: Status) {
-  let color, backgroundColor, Icon;
+  let color, backgroundColor, borderColor, Icon;
 
   switch (status) {
     case 'missed':
-      color = colors.r1;
-      backgroundColor = colors.r10;
+      color = theme.errorText;
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏some of these colors are quite different. I'm expanding the VRT tests to catch this in the future: https://github.com/actualbudget/actual/pull/1672

(yes, I know they are inconsistent with the list-view; I think the list view color change is a recent regression.. either way - we can address this independently - lets try and keep the existing lightmode with no changes unless absolutelly necessary)

Reply:
Okay, I've made exact matches for all of these. There's definitely some opportunity for creating some consistency here in a future PR.